alvr/server/cpp/platform/linux/EncodePipelineNvEnc.cpp

```diff
@@ -51,6 +51,14 @@ alvr::EncodePipelineNvEnc::EncodePipelineNvEnc(VkFrame &input_frame,
     case ALVR_CODEC_H264:
         AVUTIL.av_opt_set(encoder_ctx, "preset", "llhq", 0);
         AVUTIL.av_opt_set(encoder_ctx, "zerolatency", "1", 0);
+
+        switch (settings.m_entropyCoding) {
+        case ALVR_CABAC:
+            AVUTIL.av_opt_set(encoder_ctx, "coder", "ac", 0);
```

Comment:
First, rebase the PR on master.

It should be `AVUTIL.av_opt_set(encoder_ctx->priv_data, ...`

And are you sure `ac` `vlc` is correct?
https://github.com/FFmpeg/FFmpeg/blob/eef763c7057a7f5f4b7dae7855d07b2a6da8b537/libavcodec/vaapi_encode_h264.c#L1237

Reply:
Yes, you can find the description of those options in FFmpeg Codecs documentation under VAAPI encoders. `ac` and `cabac` correspond to CABAC and `vlc` and `cavlc` correspond to CAVLC.
https://ffmpeg.org/ffmpeg-codecs.html#VAAPI-encoders

Reply:
Should ` AVUTIL.av_opt_set(encoder_ctx->priv_data, ...` be used this way in both nvenc and vaapi code?

Reply:
I think so, yes.